config/environments/test.rb

```diff
@@ -72,4 +72,6 @@
   end
 
   config.hosts << "www.example.com"
+
+  config.redis = { url: (ENV["REDIS_URL"] || 'redis://localhost:6379/1') }
```

Comment:
Please rebase, this change should disappear 😄

Reply:
solved in 7e6b0a1